src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -340,11 +340,6 @@ class SchedulerActions(
 
   // TODO move stuff below out of the scheduler
```

Comment:
Is this TODO still valid?

Reply:
Yeah I believe it is, there are more stuff than this one